src/main/java/com/minecolonies/coremod/commands/GetColonyAndCitizen.java

```diff
@@ -122,9 +122,9 @@ else if (args.length == NAME_ARGUMENT_LENGTH && colonyId >= 0)
         {
             citizenName = args[ARGUMENT_ZERO] + " " + args[ARGUMENT_ONE] + " " + args[ARGUMENT_TWO];
             for (int i = 1
-                   ; i <= ColonyManager.getColony(colonyId).getCitizens().size(); i++)
+                   ; i <= ColonyManager.getColony(colonyId).getCitizenManager().getCitizens().size(); i++)
             {
-                if (ColonyManager.getColony(colonyId).getCitizen(i).getName() != null && ColonyManager.getColony(colonyId).getCitizen(i).getName().equals(citizenName))
+                if (ColonyManager.getColony(colonyId).getCitizenManager().getCitizen(i).getName() != null && ColonyManager.getColony(colonyId).getCitizenManager().getCitizen(i).getName().equals(citizenName))
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 207 characters long line (which is greater than 180 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00103)